test/cookbooks/cerner_splunk_ingredient_test/recipes/configure_splunk.rb

```diff
@@ -1,15 +0,0 @@
-splunk_conf 'system/indexes.conf' do
```

Comment:
This file apparently wasn't being referenced anywhere. I presume it's leftover from when I was splitting the resources into their own recipes, but I forgot to clean it up previously.